### PR TITLE
add a4a preview disclaimer

### PR DESCRIPTION
**Pull request by sebastianbenz**

### CODE REVIEW

**Comment by sebastianbenz** [comment]
@kul3r4 PTAL

File: templates/preview-a4a.html [hunk 1]
```diff
@@ -58,7 +53,8 @@
     </div>
     <article id="preview">
       <h1 id="title">{{subHeading}}</h1>
-      <p id="summary">This sample demonstrates AMP for Ads (A4A) in action.</p>
+<p class="card info">This example is to showcase the A4A creative format. Speed of actual ad delivery & rendering is not representative since this ad is not being delivered from an ad server.</p>
```
Comment (kul3r4) [inline templates/preview-a4a.html]: This example is to showcase -> This example showcases

**Comment by kul3r4** [comment]
Looks good, just a suggestion for better wording.

**Comment by sebastianbenz** [comment]
The text came from the a4a team. I'd prefer to keep it like this.

**Comment by kul3r4** [comment]
Ok! than LGTM.

**Pull request merged**